discovery-provider/src/queries/get_sitemap.py

```diff
@@ -109,12 +106,14 @@ def get_max_track_count(session: Session) -> int:
         session.query(User.handle, TrackRoute.slug)
         .join(Track, TrackRoute.track_id == Track.track_id)
         .join(User, TrackRoute.owner_id == User.user_id)
+        .join(AggregateUser, User.user_id == AggregateUser.user_id)
```

Comment:
So what it looks like is to add the same filter to the counts query that was added to the sitemaps list query?

Reply:
yup